### DISCUSSION TITLE
Add pytest fixtures (to complement pytest-django)

### DISCUSSION THREAD

**Discussion by aragilar** [encode/django-rest-framework#9506] [Ideas & Suggestions]
Currently the DRF test helpers are not available via fixtures. https://pytest-django.readthedocs.io/en/latest/ covers combining pytest and django, but does not currently support DRF. https://github.com/pytest-dev/pytest-django/issues/443 suggests this be part of DRF, would a PR adding in the new fixtures be accepted?

**Comment by auvipy** [comment]
can you share some example on what you like to add?

**Reply by aragilar** [reply]
https://github.com/pytest-dev/pytest-django/blob/main/pytest_django/fixtures.py#L400-L407 is an example of what pytest-django does, I'd envision doing something similar (while ensuring that pytest and pytest-django remain not required to use DRF or using the existing testing support).
